community.json

```diff
@@ -440,6 +440,13 @@
         "state": "inprogress",
         "url": "https://github.com/YunoHost-Apps/gitlab_ynh"
     },
+    "gitlab-runner": {
+        "branch": "master",
+        "level": 0,
```

Comment:
Not sure you have to / should set the level here (it's usually set semi-automatically via the CI result) but that shouldnt hurt

Reply:
Yes, you can definitely skip that!
It will be set with the real value on next Friday anyway :wink: